src/pykeen/models/unimodal/node_piece.py

```diff
@@ -137,6 +137,12 @@ def __init__(
                 num_tokens=num_tokens,
                 embedding_dim=embedding_dim,
             )
+        elif aggregation == "mean":
+            aggregation = torch.mean
+        elif aggregation == "sum":
+            aggregation = torch.sum
+        elif aggregation == "max":
+            aggregation = torch.max
```

Comment:
Alternative:
```suggestion
        else:
            aggregation = getattr(torch, aggregation)
```

Reply:
I like being a bit more explicit. This alternative allows the code to do some wrong things, like if you put "tan"

Reply:
It would also allow to e.g. use [logsumexp](https://pytorch.org/docs/stable/generated/torch.logsumexp.html) (= soft version of maximum) without requiring any code changes.

Reply:
I still think it's better to explicitly handle these, so we can better document what users should put in